content/applications/finance/fiscal_localizations/colombia/configuration.rst

```diff
@@ -0,0 +1,132 @@
+=============
+Configuration
+=============
```

Comment:
According to new TOC layout BE team rolled out recently, this doc should be integrated into `colombia.rst`  instead of being it's own doc.